test/support/supportedExtensions.ts

```diff
@@ -0,0 +1,277 @@
+/* tslint:disable max-line-length */
```

Comment:
If you just want to reflect the structure then use a smaller file with as many entries as necessary, not the whole file. It will be more maintainable this way.